toolkit/tools/scheduler/schedulerutils/implicitprovides.go

```diff
@@ -70,7 +70,7 @@ func replaceNodesWithProvides(res *BuildResult, pkgGraph *pkggraph.PkgGraph, pro
 
 	// If there is no clear parent node for the implicit provide error out.
 	if parentNode == nil {
-		return fmt.Errorf("unable to find suitable parent node for implicit provides (%s) with file (%s)", provides, rpmFileProviding)
+		return fmt.Errorf("failed to find suitable parent node for implicit provides (%s) with file (%s)", provides, rpmFileProviding)
```

Comment:
The words `unable` and `could not` are probably okay within an error message. But `failed` also works. So, either/or,

Reply:
Yes, trying to stay consistent in error messages shown to user